api/src/main/java/org/apache/gravitino/authorization/Privilege.java

```diff
@@ -89,9 +89,15 @@ enum Name {
     CREATE_ROLE(0L, 1L << 16),
     /** The privilege to grant or revoke a role for the user or the group. */
     MANAGE_GRANTS(0L, 1L << 17),
-    /** The privilege to create a model */
+    /** The privilege to register a model */
+    REGISTER_MODEL(0L, 1L << 18),
+    /** The privilege to create a model. This is deprecated. Please use REGISTER_MODEL */
     CREATE_MODEL(0L, 1L << 18),
```

Comment:
Shall we use a new `bit` to represent the newly added privilege? What will happen when the `bits` conflict?